dotcom-rendering/src/model/buildLightboxImages.ts

```diff
@@ -68,25 +42,18 @@ const decideImageId = (image: ImageForLightbox): string | undefined => {
 			// E.g.
 			// https://static.guim.co.uk/sys-images/Guardian/Pix/pictures/2015/5/26/1432666797165/59de49e2-553f-4b52-b7ac-09bda7f63e4b-220x132.jpeg
 			// This bit                                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-			return url.pathname.split('/').reverse()[0];
+			return url.pathname.split('/').at(-1);
 		default:
-			return;
+			return undefined;
 	}
 };
 
 const buildLightboxImage = (
 	element: ImageBlockElement | CartoonBlockElement,
-): ImageForLightbox | undefined => {
-	const cartoons =
-		element._type ===
-		'model.dotcomrendering.pageElements.CartoonBlockElement'
-			? element.variants.flatMap((v) => v.images)
-			: [];
-	const images =
-		element._type === 'model.dotcomrendering.pageElements.ImageBlockElement'
-			? element.media.allImages
-			: [];
-	const allImages = images.concat(cartoons);
+): Omit<ImageForLightbox, 'position'> | undefined => {
```

Comment:
We want to omit the `position` value, because it can only be known once the entire array has been generated.